todo_app/templates/index.html

```diff
@@ -1,4 +1,5 @@
 {% extends "layout.html" %}
+{% from "item_list.html" import item_list as item_list %}
```

Comment:
You don't need the `as` modification as you are not altering the name of the macro. 

```suggestion
{% from "item_list.html" import item_list %}
```